catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
+per provider.
 
 # standardized_popularity
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
+Normalized popularity, a calculated column. Only available for providers that
+have popularity data.
 
 ## Normalization and Validation
 
-## Existing Data Inconsistencies
+The value is updated monthly during the data refresh process.
 
 # duration
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The duration of the main audio file in milliseconds.
 
 # bit_rate
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The bit rate of the main audio file.
 
 # sample_rate
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The sample rate of the main audio file.
 
 # genres
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+List of genres associated with the audio.
 
 # alt_files
 
 ## Description
 
+The list of alternative file details for the audio (different formats/ quality).
+
 ## Object Shape
 
-## Selection Criteria
+JSONb array of dictionaries:
 
-## Normalization and Validation
+`[{"url": "http://example.com/audio.mp3", "filesize": 123456, "bit_rate": 128, "sample_rate": 44100}]`
 
-## Existing Data Inconsistencies
+- `url` (string): the direct URL of the alternative file.
+- `filesize` (integer): the size of the alternative file in bytes.
+- `filetype` (string): the file type (extension) of the alternative file.
+- `bit_rate` (integer): the bit rate of the alternative file.
+- `sample_rate` (integer): the sample rate of the alternative file.
+- `duration` (integer, optional): the duration of the alternative file in
+  milliseconds.
 
 # audio_set
 
 ## Description
 
-## Object Shape
+The information about the audio set (collection, album) that the audio belongs
+to.
 
-## Selection Criteria
+## Object Shape
 
-## Normalization and Validation
+JSON object:
+`{"title": "Audio Set Title", "foreign_landing_url": "http://example.com", "thumbnail": "http://example.com/thumbnail.jpg", "creator": "Creator Name", "creator_url": "http://example.com/creator", "foreign_identifier": "123456"}`
 
-## Existing Data Inconsistencies
+- title (string): the title of the audio set.
+- foreign_landing_url (string): the URL of the audio set on the source site.
+- thumbnail (string): the URL of the thumbnail image for the audio set.
+- creator (string): the name of the creator of the audio set.
+- creator_url (string): the URL of the creator's page, usually on the source
+  site.
+- foreign_identifier (string): the unique identifier for the audio set on the
+  source site. This identifier is saved in the `audio_set_foreign_identifier`
+  field of the TSV and catalog audio table.
```

Comment:
Nit: can you wrap these items in backticks as well, to make them monospace like the section above it?